agent/eventhandler/task_handler_types.go

```diff
@@ -90,13 +90,18 @@ func (event *sendableEvent) taskShouldBeSent() bool {
 			// change to be sent to ECS.
 			return true
 		}
-		for _, ma := range container.GetManagedAgents() {
-			if ma.SentStatus < ma.Status {
-				return true
-			}
-		}
 	}
-	return false
+
+	// Managed agent event should be sent
+	// TODO check get sent status for managed agents
+	// note `make static-check` required this.
```

Comment:
do we need these comments?

Reply:
We do. This section wouldn't make sense with what `make static-check` required. And it will change when I add the sent status.